src/content/app/genome-browser/components/drawer/drawer-views/sequence-view/GeneSequenceView.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { noop } from 'lodash';
+
+import * as urlFor from 'src/shared/helpers/urlHelper';
+
+import RadioGroup, {
+  RadioOptions
+} from 'src/shared/components/radio-group/RadioGroup';
+
+import { GeneSummaryQueryResult } from 'src/content/app/genome-browser/state/api/queries/geneSummaryQuery';
+
+import styles from './SequenceView.scss';
+
+type Gene = {
+  slice: GeneSummaryQueryResult['gene']['slice'];
+};
+
+type Props = {
+  gene: Gene;
+};
+
+export const GeneSequenceView = (props: Props) => {
+  const sequenceType = 'genomicSequence';
+  const checksum = props.gene.slice.region.sequence.checksum;
+  const start = props.gene.slice.location.start;
+  const end = props.gene.slice.location.end;
+  const sequenceURL = urlFor.refget({ checksum, start, end });
+
+  const radioOptions: RadioOptions = [
+    {
+      value: 'genomicSequence',
+      label: 'Genomic sequence'
+    }
+  ];
+
+  return (
+    <div className={styles.layout}>
+      <div>
+        <div>XXXX bp</div>
```

Comment:
`bp` will have to be styled differently from the count. Is that for a different PR?

Reply:
Yes it is for [ENSWBSITES-1547](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1547)